gems/business_time/0.13/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  - name: date
+  - name: time
+  - name: monitor
+  - name: singleton
+  - name: logger
+  - name: mutex_m
+  - name: minitest
+  - name: json
+  - name: benchmark
+  - name: digest
+  - name: set
```

Comment:
I guess most of all dependencies are activesupport's. If so, they are unnecessary here.

Reply:
I removed it because the dependency on activesupport alone will provide everything.

Reply:
Only `set` was left.